pkg/sqlite/load.go

```diff
@@ -803,5 +803,23 @@ func (s *sqlLoader) addBundleProperties(tx *sql.Tx, bundle *registry.Bundle) err
 		}
 	}
 
+	// Add label properties
+	if csv, err := bundle.ClusterServiceVersion(); err == nil {
+		for k, v := range csv.GetLabels() {
```

Comment:
I don't think Kube labels are the place to represent operator labels. That makes operator labels subject to the same name restrictions as Kube labels. We also don't expect operator labels to necessarily be tied to CSV selection/aggregation on-cluster.

Reply:
If it is not in the labels then where do you think it should be? Annotations? I use Labels because the name of the type is label. That only makes sense to have it in labels if we can it `label` type. Plus, any CSV selection/aggregation will need to select labels based on keys. As long as we preserve the `olm.labels` key to use for this purpose then we should be fine.

Reply:
@ecordell FYI